devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -215,24 +215,47 @@ def getTemperature(value) {
 }
 
 private Map getBatteryResult(rawValue) {
-	log.debug 'Battery'
+	log.debug "Battery rawValue = ${rawValue}"
 	def linkText = getLinkText(device)
-    
-    def result = [
-    	name: 'battery'
-    ]
-    
+
+	def result = [
+		name: 'battery',
+		value: '--'
+	]
+
 	def volts = rawValue / 10
-	def descriptionText
-	if (volts > 3.5) {
-		result.descriptionText = "${linkText} battery has too much power (${volts} volts)."
-	}
+
+	if (rawValue == 255) {}
```

Comment:
Previously this sensor wasn't checking to make sure the battery attribute was valid (!= 0xFF)